PubNub/Data/Managers/PNPublishSequence.m

```diff
@@ -369,30 +376,6 @@ - (void)subscribeOnNotifications {
 #endif // TARGET_OS_OSX
 }
 
-- (void)unsubscribeFromNotifications {
```

Comment:
I saw them, but there is no conflicts in this file.
I will be able to resolve them after review process will be marked as resolved (because conflict resolve locally and pushed directly to MPR target branch).